code/modules/mob/new_player/new_player.dm

```diff
@@ -87,6 +87,9 @@
 	..()
 
 	statpanel("Lobby")
+	if(client.statpanel == "Status" && SSticker)
+		if((SSticker.current_state == GAME_STATE_PREGAME)
```

Comment:
```suggestion
		if(SSticker.current_state == GAME_STATE_PREGAME)
```

Reply:
did it on my phone sorry bout that, fixing